src/fmtspec.jl

```diff
@@ -9,25 +9,26 @@
 #  width ::= <integer>
 #  prec  ::= <integer>
 #  type  ::= 'b' | 'c' | 'd' | 'e' | 'E' | 'f' | 'F' | 'g' | 'G' |
-#            'n' | 'o' | 'x' | 'X' | 's'
+#            'n' | 'o' | 'x' | 'X' | 's' | 'S'
```

Comment:
I really don't want any non-standard characters added, `'s'` works fine for outputting symbols already.